dev-client/src/auth/index.ts

```diff
@@ -80,13 +80,16 @@ async function exchangeToken(
   identityJwt: string,
   provider: Omit<AuthProvider, 'google'> | `google-${'ios' | 'android'}`,
 ) {
+  let body: any = {
+    provider,
+    jwt: identityJwt,
+  };
+  if (provider === 'apple') {
+    body.client_id = Constants.expoConfig!.ios?.bundleIdentifier;
```

Comment:
Suggestion: Could this use `apple.clientId` from line 46 above?
```suggestion
    body.client_id = configs.apple.clientId;
```

Reply:
@knipec what about `body.client_id = APP_CONFIG.appleClientId;`?